apps/nerves_hub_device/lib/nerves_hub_device_web/channels/device_channel.ex

```diff
@@ -121,12 +140,14 @@ defmodule NervesHubDeviceWeb.DeviceChannel do
     # If we get here, the device is connected and high probability it receives
     # the update message so we can Audit and later assert on this audit event
     # as a loosely valid attempt to update
-    AuditLogs.audit!(deployment, socket.assigns.device, :update, %{
-      from: "broadcast",
-      send_update_message: true
-    })
+    if not socket.assigns.download_uuid do
```

Comment:
```suggestion
    if is_nil(socket.assigns.download_uuid) do
```

`not` will require an explicit true/false condition for comparison. `nil` or a binary will fail it